src/test/kotlin/com/pubnub/extension/JsonElementTest.kt

```diff
@@ -1,67 +1,76 @@
 package com.pubnub.extension
 
+import com.google.gson.Gson
+import com.google.gson.JsonArray
 import com.google.gson.JsonElement
 import com.google.gson.JsonObject
 import com.google.gson.JsonPrimitive
 import com.pubnub.api.CommonUtils.generatePayload
 import com.pubnub.api.crypto.CryptoModule
 import com.pubnub.api.managers.MapperManager
-import com.pubnub.contract.gson
 import org.junit.jupiter.api.Assertions.assertEquals
-import org.junit.jupiter.api.Test
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.Arguments
 import org.junit.jupiter.params.provider.MethodSource
 
 class JsonElementTest {
 
     companion object {
+        val gson = Gson()
+
         @JvmStatic
         fun cryptoModuleConfiguration(): List<Arguments> {
             val cipherKey = "enigma"
             return listOf(
                 Arguments.of(
-                    "Hello world.",
+                    JsonPrimitive("Hello world."),
                     false,
                     "X/wLZqR/I+4eU7WwtKycTwy9+L9e7m2BhklK9ZQWtwI=",
                     CryptoModule.createLegacyCryptoModule(cipherKey = cipherKey, randomIv = true),
                     generatePayload()
```

Comment:
I guess we can remove 'generatePayload()' from all Arguments.

Reply:
Done